spec/rails/integration/forms_spec.rb

```diff
@@ -23,7 +23,7 @@
     end
 
     it "should include the hidden authenticity token" do
-      expect(html).to include '<input type="hidden" name="authenticity_token" value="AUTH_TOKEN" />'
+      expect(html).to have_selector('input[type="hidden"][name="authenticity_token"][value="AUTH_TOKEN"]', visible: :hidden)
```

Comment:
Just curious as to why you had to change this? Was it just for consistency? No issue with the change since if it is regarding the latter, the rest of the file uses it.

Reply:
>  Does the commit message need to be updated?

Yes, I've used two different approaches. This one is compatible with both rails 6 and 7. Will update the commit message

Reply:
Great, that's what I figured. Had to do with compatibility. Thank you for updating the commit message as its much clearer now!